config/sets/doctrine-orm-213.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\MethodCall\RenameMethodRector;
+use Rector\Renaming\Rector\Name\RenameClassRector;
+use Rector\Renaming\ValueObject\MethodCallRename;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(
+        RenameMethodRector::class,
+        [
+            // @see https://github.com/doctrine/orm/issues/9875
```

Comment:
This link goes to some parent class renames. Is it correct?

Reply:
The deprecated class and functions where also linking to this page so I thought it was appropriate

Reply:
That issues seems WIP and unclear, what exactly is handles to me :)

This Rector rule renames method, so it should link commit/PR that renames method too. Could you link it?

Reply:
I added correct links

Reply:
Perfect, thank you :+1: